src/main/java/io/cryostat/net/NetworkModule.java

```diff
@@ -88,12 +93,27 @@ static NetworkResolver provideNetworkResolver() {
         return new NetworkResolver();
     }
 
+    @Provides
+    @Named(MAX_TARGET_CONNECTIONS)
+    static int provideMaxTargetConnections(Environment env) {
+        return Integer.parseInt(env.getEnv(MAX_TARGET_CONNECTIONS, "-1"));
+    }
+
+    @Provides
+    @Named(MAX_TARGET_TTL)
+    static Duration provideMaxTargetTTL(Environment env) {
+        return Duration.ofSeconds(Integer.parseInt(env.getEnv(MAX_TARGET_TTL, "10")));
```

Comment:
As it is now, the TTL will either be 5 or 10 seconds depending on whether the environment variable exists or not. 

Is this because the previous TTL of 90 seconds was overkill?

Reply:
It's set to 5 in integration tests, and it defaults to 10 if the user/deployment doesn't set their own value. I set it to a non-default value in testing mostly just to exercise that a bit and make sure setting a non-default value doesn't blow anything up.

I feel 90 seconds was overkill in retrospect, although this is really just my feeling. It's best not to hold the connections open for too long/indefinitely since there may be arbitrarily many of them, but it's also preferable for performance reasons to not open a new connection for every single request or else the end user will likely experience bad latency and UI responsiveness, so there is some balance to be found.

Ideally this should probably be defaulted to some value that minimizes the occurrence of an interactive web-client user incurring JMX reconnections while they click through some typical workflows at a typical pace, but I don't really know what that value would be for certain.

Reply:
Makes sense, thanks.